pypowerwall/local/pypowerwall_local.py

```diff
@@ -181,11 +183,11 @@ def poll(self, api: str, force=False, recursive=False, raw=False) -> Optional[di
                 payload = r.raw.data
             else:
                 payload = r.text
-            try:
-                payload = json.loads(payload)
-            except Exception as exc:
-                log.error(f'Unable to parse payload as JSON: {exc}')
-                return None
+                try:
+                    payload = json.loads(payload)
+                except Exception as exc:
+                    log.error(f'Unable to parse payload as JSON: {exc}')
+                    return None
```

Comment:
The raw payload (protobuf binary) needs to be cached and returned without any processing.

Reply:
Oops, sorry, my bad! Good catch!

Reply:
Nice!